ui/segment.reel/segment.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+    <link rel="stylesheet" type="text/css" href="segment.css">
+    <script type="text/montage-serialization">
+    {
+        "owner": {
+            "properties": {
+                "element": {"#": "Segment"}
+            }
+        },
+        "text": {
+            "prototype": "ui/text.reel",
+            "properties": {
+                "element": {"#": "text"}
+            },
+            "bindings": {
+                "value": {"<-": "@owner.content.label"}
+            }
+        }
+    }
+    </script>
+</head>
+<body>
+<div data-montage-id="Segment" class="montage-Segment">
+    <span data-montage-id="text"></span>
+</div>
+</body>
+</html>
```

Comment:
It's a webstorm thing, this way things inside body isn't indented too deeply, will fix